lib/src/async.dart

```diff
@@ -9,7 +9,7 @@ part of 'hooks.dart';
 ///   * [Future], the listened object.
 ///   * [useStream], similar to [useFuture] but for [Stream].
 AsyncSnapshot<T> useFuture<T>(Future<T> future,
-    {T initialData, bool preserveState = true}) {
+    {T? initialData, bool preserveState = true}) {
```

Comment:
initialData probably should be `required T initialData`

It's a breaking change, but matches with how `FutureProvider`/`StreamProvider` migrated

Reply:
would you like to keep it as a named parameter or what do you think about either both `future` and `T initialData` as positioned parameter or both as named?